conf/nginx/via/direct_proxy.conf

```diff
@@ -21,7 +21,8 @@ rewrite ^/proxy/static/(.*)$ $1? break;
 # Add back on the args which we stripped above so we don't get them twice
 proxy_pass $uri$is_args$args;
 
-proxy_redirect ~^(.*)$ $original_scheme://$http_host/proxy/static/$1;
+proxy_redirect ~^(https?:\/\/)(.*)$ $original_scheme://$http_host/proxy/static/$1$2;
+proxy_redirect ~^(.*)$ $original_scheme://$http_host/proxy/static/https://$proxy_host$1;
```

Comment:
The first `proxy_redirect` regex matches absolute URLs (`http://` or `https://`). Only if that regex does not match the URL will NGINX then look at the second `proxy_redirect` regex which matches anything (and therefore in practice only matches relative URLs).

Note that hard-coded `https://` in the second line